### ISSUE TITLE
Force delete of a Rock-on

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-doc#276]
We have in place, for some time now, a command line script that does a force delete of a Rock-on by name via the command line:
```
/opt/rockstor/bin/delete-rockon 
Delete metadata, containers and images of a Rock-on
	Usage: /opt/rockstor/bin/delete-rockon <rockon name>
```
Although there are plans to integrate this forced/emergency rock-on removal method into our Web-UI, say via an advanced option / link possibly exposed after a set period of Rock-on transition post expected completion; or just in an advanced menu somewhere, it would still be useful to have this command line capability documented. Especially for older installs that pre-date our pending Web-UI improvements re dealing with problematic/stuck Rock-on install/un-install etc.

The proposed document page would be:
https://rockstor.com/docs/docker-based-rock-ons/overview.html

**Comment by phillxnet** [comment]
An example explanation of this facilities use is available in the following forum post:
https://forum.rockstor.com/t/how-to-fresh-reinstall-openvpn-rock-ons/4270/4

**Comment by FroggyFlox** [comment]
I'm currently working on this issue.

I was thinking of doing the following: at the end of the Rock-ons overview page, include a new section entitled "Uninstalling a Rock-on". In this section, describe how to uninstall a Rock-on through the web-UI, emphasizing the following points:

- Ease of doing so: click the button, confirm, and that's it.
- Uninstalling a Rock-on will remove its related database entries but **not delete any data on the disk**. It is thus an easy way to update the container to the newest image available.

In the unlikely event one cannot go through this process via Rockstor's webUI, we provide the `delete-rockon` script that can be used to uninstall a rock-on from the command line (and then describe how to use it).

@phillxnet, do you think it sounds like a good idea to you?
Actually, I just realized that this would also fix #274, although the latter would still require its own section so that users can easily search for it and find it.
